hack/build/setup-go.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# script to setup go version with gimme as needed
+# MUST BE RUN FROM THE REPO ROOT DIRECTORY
+
+# read go-version file unless GO_VERSION is set
+GO_VERSION="${GO_VERSION:-"$(cat .go-version)"}"
+
+# only setup go if we haven't set FORCE_HOST_GO, or `go version` doesn't match
+# go version output looks like:
+# go version go1.14.5 darwin/amd64
```

Comment:
something somewhere is outputting the go version, can't seem to pinpoint where :/

Reply:
it's related to gimme, I've failed to silence it properly
gimme has an env to disable this, but it only works if used during the first time setting up that version, they don't regenerate the .evn file. not sure what the best fix is yet.

Reply:
it is mostly cosmetic though

Reply:
fixed in https://github.com/kubernetes-sigs/kind/pull/1769/commits/83d19a41281d0eeea0871c94039a0a3fc7c73bd3